nano/lib/numbers.cpp

```diff
@@ -61,6 +62,16 @@ std::string nano::public_key::to_account () const
 	return result;
 }
 
+nano::public_key::public_key () :
+	uint256_union{ 0 }
+{
+}
+
+nano::public_key::public_key (std::nullptr_t) :
```

Comment:
I thought we were supposed to remove the nullptr_t constructor to avoid the 0 constructor problem?

Reply:
No, I understood we leave this constructor but make the default one behave as if it was constructed with a `0` argument.